roles/edpm_network_config/defaults/main.yml

```diff
@@ -18,6 +18,7 @@
 # All variables intended for modification should be placed in this file.
 
 # All variables within this role should have a prefix of "edpm_network_config"
+edpm_network_config_role: os-net-config
```

Comment:
As this isn't an actual role name, I not use the _role suffix. Perhaps call it edpm_network_config_tool. I'd also document the choices as either os-net-config or nmstate.

Reply:
ack.